hacking.md

```diff
@@ -0,0 +1,303 @@
+# Developing SPO
+This document describes how to build, install and test SPO for development
+purposes. It is not exhaustive - knowledge of how an operator works is
+presumed and PRs are always welcome.
+
+## Building SPO locally
+Even though SPO is a Kubernetes operator and as such is normally not meant to
+be used locally, but rather deployed in a cluster from pre-built images,
+it's often useful to be able to run a quick build or a test on your local machine.
+
+Depending on what OS and version you are developing on and what features you
+want to build with, you might need to either install extra dependencies or
+disable features that would require them.
+
+There are currently two optional features at build time
```

Comment:
libseccomp is a non-optional dependency which is required for building the operator, too. I mean, yeah we disable it for Darwin build environments, but it truly degrades the bpf recorder and enricher feature.

Reply:
sorry, but I'm not sure what do you want me to fix here? The doc describes that the eBPF recorder is optional, not seccomp.

Feel free to provide some proposal of wording that you think is better.

Reply:
oh, do you want me to add that `libseccomp` is always required? Sorry, I'm a little slow today :-)

Reply:
Yeah but do we wanna mention that libseccomp is required for building the operator as well?